src/components/Calculator.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Output from './Output';
+import ButtonsBox from './ButtonsBox';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please kindly do not disable linters errors, fix linters helps programmers to write good and clean codebase. Kindly fix your errors 🙃 😉